src/ngrx/11-create-reducer/contact.ts

```diff
@@ -1,3 +1,4 @@
+// Note: This example code is not part of our application repo or solution
```

Comment:
Normally when we header a file with a comment, we have a newline between the code and the comment. Should we add one for this kind of comment?